### ISSUE TITLE
Wrong version used for alpine image of packer light

### ISSUE THREAD

**Issue by saez0pub** [hashicorp/docker-hub-images#83]
You pushed a wrong version of alpine : 
```
$ docker pull hashicorp/packer:latest
latest: Pulling from hashicorp/packer
Digest: sha256:001e6d7f728d0a672c4b695c60b26b56402d430e528f2c3995b69949e5cc4a0c
Status: Image is up to date for hashicorp/packer:latest
$ docker pull alpine:latest
latest: Pulling from library/alpine
Digest: sha256:ca1c944a4f8486a153024d9965aafbe24f5723c1d5c02f4964c045a16d19dc54
Status: Image is up to date for alpine:latest
$ docker run --rm --entrypoint /usr/bin/env hashicorp/packer:latest cat /etc/alpine-release
3.7.0
$ docker run --rm --entrypoint /usr/bin/env alpine:latest cat /etc/alpine-release
3.10.0
```

**Comment by timothyclarke** [comment]
I've just hit the same issue, but I disagree on your statement. 
It's the implementation / lazy programming of the dockerfile that is at fault. The issue is the Dockerfile `FROM alpine:latest` It should be a fixed version that gets bumped per release. unless someone at hashicorp is watching the library/alpine image and triggering a rebuild each time the alpine container is released

**Comment by gitirabassi** [comment]
agreed!!

**Comment by SwampDragons** [comment]
This is fixed now for the v1.5.2 release, and I've made a change to the deploy script we use to force updates to keep it on the actual latest container.
